build/charts/antrea/crds/traceflow.yaml

```diff
@@ -201,14 +201,269 @@ spec:
                       type: string
                     length:
                       type: integer
+                      minimum: 0
```

Comment:
I don't know if we should make any modification to the old schema, even for the `status`.

Reply:
Removed.